internal/applier/bundledeployment_applier.go

```diff
@@ -24,21 +24,29 @@ func NewBundleDeployment(po *platformv1alpha1.PlatformOperator, image string) *r
 }
 
 // buildBundleDeployment is responsible for taking a name and image to create an embedded BundleDeployment
-func buildBundleDeployment(image string) rukpakv1alpha1.BundleDeploymentSpec {
-	return rukpakv1alpha1.BundleDeploymentSpec{
+func buildBundleDeployment(image string) rukpakv1alpha2.BundleDeploymentSpec {
+	return rukpakv1alpha2.BundleDeploymentSpec{
 		ProvisionerClassName: plainProvisionerID,
```

Comment:
Looks like ProvisionerClassName should be `registryProvisionerID`. The bundle provisioner id would go in here.

Reply:
I actually think platform operators was specifically designed not to work with registryv1 bundles. Did we lose the plain provisioner in the latest revision?

Reply:
Plain provisioner does exist. I noticed BundleSpec.provisionerClassName to be registry. Which is what is supposed to be specified here. Basically `spec.provisionerClassName` is alpha2 should be equal to `template.spec.provisionerClassName`

Reply:
I tested this a little bit and you were correct. We actually don't have support for configuring the provisioner in PO and the only test examples we have support for are registryv1, so the right thing to migrate here is to use the registryv1 provisioner. I've updated accordingly.